cg_hermes/config/balsamic.py

```diff
@@ -538,6 +566,10 @@
     frozenset(RAW_TAGS["ascat.copynumber.txt.gz"]): {"is_mandatory": True},
     frozenset(RAW_TAGS["tumor.vcf2cytosure.cgh"]): {"is_mandatory": True},
     frozenset(RAW_TAGS["normal.vcf2cytosure.cgh"]): {"is_mandatory": True},
+    frozenset(RAW_TAGS["cov.bed.gz"]): {"is_mandatory": True},
```

Comment:
I'm also aware that in this case we have 2 sets of files for T + N analysis, and with no unique ID in the filename like "normal" and "tumor" but simply the dynamic LIMS-ID. I don't know if this would be unique for housekeeper files for balsamic. But do you think it will be an issue for storing? Basically will it be capable of storing both sets of GENS-files? : )

Reply:
OK, I understand your concern. Looking at the code It should find both files and save them with the respective sample ID. I will check it before resolving this comment :)